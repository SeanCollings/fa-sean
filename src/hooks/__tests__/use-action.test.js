import { useDispatch } from 'react-redux';
import { useActions } from '../use-actions';

jest.mock('react-redux');

describe('use action hook', () => {
  useDispatch.mockReturnValue(jest.fn());

  test('should return', () => {
    const action = useActions();
    expect(action).toMatchSnapshot();
  });
});
