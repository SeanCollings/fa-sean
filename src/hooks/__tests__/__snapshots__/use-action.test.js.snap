// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`use action hook should return 1`] = `
Object {
  "updateFilter": [Function],
}
`;
